docs/content/developers/building-contributing.md

```diff
@@ -67,21 +67,28 @@ brew link --force ddev
 
 ### Installing a Downloaded Binary in the `$PATH`
 
-Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `~/bin` in `$PATH`, you can create the folder and add it to your path in `~/.bashrc` with these commands:
+Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `~/bin` in `$PATH`, you can create the folder and add it to your path.
 
+Add `~/bin` to your path, by adding this in your `~/.bashrc` file:
 ```
-mkdir -p ~/bin
 export PATH="~/bin:$PATH"
```

Comment:
This should be `$HOME/bin`, not `~/bin`, because:

- #6722

Let me fix it.

Reply:
Great, feel free to update it.